decidim-proposals/app/views/decidim/proposals/proposals/_tags.html.erb

```diff
@@ -0,0 +1,25 @@
+<figure>
+  <figcaption class="text-uppercase">
```

Comment:
@Crashillo @decidim/lot-core the `<figure>` element is not holding any image or diagram, is this an accepted use?

[The figure & figcaption elements | HTML5 Doctor](http://html5doctor.com/the-figure-figcaption-elements/)

This is taken from the `decidim_app-design` template: https://github.com/decidim/decidim/blob/a6fa0b41a9a118a17e485fd754dce07399c43327/decidim_app-design/app/views/public/proposals/proposal-view.html.erb#L87

Reply:
I got it from [here](https://stackoverflow.com/a/35019847/5020256). I thought it was cool.
You can get the same with a nested list.

Reply:
Ok, only wondering.

It can be more semantic:

```
<section>
  <header>Filed as</header>
  <ul>
    <li>item 1</li>
  </ul>
</section>
```